manifests/s/Stoplight/Studio/2.8.1/Stoplight.Studio.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Stoplight.Studio
+PackageVersion: 2.8.1
+PackageLocale: en-US
+Publisher: Stoplight
+PublisherUrl: https://stoplight.io/
+PublisherSupportUrl: https://support.stoplight.io/
+PrivacyUrl: https://stoplight.io/privacy
+Author: Stoplight
+PackageName: Studio

```

```suggestion
PackageName: Stoplight Studio
```
This should match control panel as closeley as possible